wls-gui-wahllokalsystem/src/components/wahlvorbereitung/beanstandeteWahlbriefe/TheBeanstandeteWahlbriefeBeschlussergebnis.vue

```diff
@@ -5,41 +5,41 @@
       <v-table>
         <tbody>
           <tr>
-            <td>
-              <b>Zugelassen</b>
-            </td>
+            <td class="font-weight-bold">Zugelassen</td>
             <td
-              v-for="(value, index) in sumGueltig"
+              v-for="(value, index) in summeGueltigerWahlbriefe"
               :key="index"
+              class="font-weight-bold"
             >
               {{ value }}
             </td>
           </tr>
           <tr>
-            <td><b>Nicht zugelassen</b></td>
+            <td class="font-weight-bold">Nicht zugelassen</td>
             <td
-              v-for="(value, index) in sumUngueltig"
+              v-for="(value, index) in summeUngueltigerWahlbriefe"
               :key="index"
+              class="font-weight-bold"
             >
               {{ value }}
             </td>
           </tr>
-          <tr>
-            <td><b>Zurückweisungsgrund</b></td>
+          <tr class="bg-grey-lighten-3 border-b border-grey-lighten-1 mb-2">
```

Comment:
diese stylings haben bei mir gar keinen effekt, was machen die? können wir die ggf weglassen?
```suggestion
          <tr class="bg-grey-lighten-3">
```

Reply:
[78f7990](https://github.com/it-at-m/Wahllokalsystem/pull/1651/commits/78f7990d1de8e25b4fcb2385b7e48cd9defb6272)